### PR TITLE
Add lynis baselines for SLES15 SP4 and fix await_install() time out issue on s390x

### PR DESCRIPTION
**Pull request by lilyeyes**
Add lynis baselines for SLES15 SP4 and fix await_install() time out issue on s390x

- Related ticket:
  https://progress.opensuse.org/issues/99447
  https://progress.opensuse.org/issues/99291
- Needles: none
- Verification run:
  VR for lynis on all arches, the failed case can be tracked by bsc#1189415.
  https://openqa.suse.de/tests/7260453 (aarch64 gnome)
  https://openqa.suse.de/tests/7260413 (aarch64 textmode)
  https://openqa.suse.de/tests/7260414 (ppc64 gnome)
  https://openqa.suse.de/tests/7260415 (ppc64 textmode)
  https://openqa.suse.de/tests/7260416 (s390x gnome)
  https://openqa.suse.de/tests/7260416 (s390x textmode)
  https://openqa.suse.de/tests/7260418 (x86_64 gnome)
  https://openqa.suse.de/tests/7260419 (x86_64 textmode)

  VR for create_hdd_textmode on s390x:
  https://openqa.suse.de/tests/7259945#details (passed)
  https://openqa.suse.de/tests/7259946#details (passed)

### CODE REVIEW

**Comment by jouyingbin** [comment]
@lilyeyes The baselines are confirmed with the developer and PM in advance, right? or it will be changed later.

Others are LGTM.

**Comment by lilyeyes** [comment]
> @lilyeyes The baselines are confirmed with the developer and PM in advance, right? or it will be changed later.
> 
> Others are LGTM.

Good question, I have checked the baselines by myself, there are only minor changes, then no need developer's review.

File: tests/installation/await_install.pm [hunk 1]
```diff
@@ -109,7 +109,7 @@ sub _set_timeout {
 sub run {
     my $self = shift;
     # NET isos are slow to install
-    my $timeout = 2000;
+    my $timeout = 2400;
```
Comment (b10n1k) [inline tests/installation/await_install.pm:112]: this is too generic magic. based on the description what you want is to fix an issue in a particular aarch. Why not `my $timeout = is_s390x ? 2400 : 2000;` or something alike

Comment (lilyeyes) [reply tests/installation/await_install.pm:112]: Good pointing, adding 400s directly is simple and doing no harm to other arches so I chose this way.

**Comment by lilyeyes** [comment]
> > @lilyeyes The baselines are confirmed with the developer and PM in advance, right? or it will be changed later.
> > Others are LGTM.
> 
> Good question, I have checked the baselines by myself, there are only minor changes, then no need developer's review.

Done, and the verify run is good: https://openqa.suse.de/tests/7273661#

File: tests/installation/await_install.pm (hunk 1 above)
Comment (lilyeyes) [reply tests/installation/await_install.pm]: Done, and the verify run is good: https://openqa.suse.de/tests/7273661#

**Review by jouyingbin — APPROVED** [review]
LGTM, thanks for update.

**Review by rfan1 — APPROVED** [review]
LGTM

File: tests/installation/await_install.pm (hunk 1 above)
Comment (b10n1k) [reply tests/installation/await_install.pm]: i want to point that in general is not a good idea. other than that no issue.

**Review by Amrysliu — APPROVED** [review]

**Pull request merged**